examples/unresolved-cfd-dem/boycott-effect/boycott-effect.prm

```diff
@@ -199,7 +199,7 @@ subsection boundary conditions
 
   subsection lagrangian physical properties
     set gx                       = 0.0
-    set gy                       = -9.81
+    set gy                       = 0.0
```

Comment:
I am not sure about this, I think the gravity should be on here and not off no? This is the CFD-DEM simulation no?

Reply:
True.  I wanted to change the documentation not the actual prm. Just fixed it.